product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_3_7_Proxy_replace_targetDefinedThroughXpathSourceInlineGovReg.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy name="1_6_3_7_Proxy_replace_targetDefinedThroughXpathSourceInlineGovReg" startOnLoad="true" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
```

Comment:
Let's remove the additional '_' after "replace" for naming convention

Reply:
This is done.